cookbook/web-server/franken-php.rst

```diff
@@ -0,0 +1,63 @@
+Running Franken PHP
```

Comment:
would call this FrankenPHP via Docker

as you can use also frankenphp standalone binary which is different to use. Same for the file name.